packages/sdk/src/swap/sdk.ts

```diff
@@ -72,9 +74,22 @@ export class SwapSDK {
 
   executeSwap(params: ExecuteSwapParams): Promise<ContractReceipt> {
     assert(this.signer, 'No signer provided');
-    return executeSwap(params, {
-      network: this.network,
+    return executeSwap(params, { network: this.network, signer: this.signer });
```

Comment:
would prefer returning the hash here too to make things consistent with the other methods